example.main.scss

```diff
@@ -2,7 +2,7 @@
    INUITCSS
    ========================================================================== */
 
-/*!*
```

Comment:
Why not using this

`/*! inuitcss, by @csswizardry | github.com/inuitcss | inuitcss.com */`

instead of remove `!`